python/cudf/cudf/api/types.py

```diff
@@ -126,6 +126,7 @@ def is_scalar(val):
     return (
         isinstance(val, DeviceScalar)
         or isinstance(val, cudf.Scalar)
+        or isinstance(val, cudf.core.tools.datetimes.DateOffset)
```

Comment:
What case does this handle? Not sure I would think of a `DateOffset` as a scalar.

Reply:
We need to treat `DateOffset` as a scalar for the purpose of binops between Frames and DateOffset:
```
cudf.Series(['2000-01-01'], dtype='datetime64[ns]') + cudf.DateOffset({'days': 0})
```
From that perspective it certainly is a scalar-like object since it represents a single offset. I don't know if that necessarily means it should be generally treated as a scalar by APIs like `is_scalar`, are there contexts where it doesn't make sense to treat it as a scalar?

Reply:
actually looks like pandas treats it as a scalar as well. 

```
>>> pd.api.types.is_scalar(pd.DateOffset(days=1))
True
```